src/main/java/net/dzikoysk/funnyguilds/system/ban/BanSystem.java

```diff
@@ -7,14 +7,15 @@
 
     private static BanSystem instance;
 
-    public BanSystem() {
+    private BanSystem() {
         instance = this;
     }
 
     public static BanSystem getInstance() {
         if (instance == null) {
             new BanSystem();
```

Comment:
To powinno się setować `instance` do nowego `BanSystem`, bo potem i metoda zwróci `null`a

Reply:
No, ale ogółem, na pewno jeszcze w tym tygodniu będę miał coś do naprawy, to wtedy się to wszystko o czym piszesz powprowadza